big_tests/tests/graphql_metric_SUITE.erl

```diff
@@ -126,6 +140,26 @@ get_by_name_global_erlang_metrics(Config) ->
     %% Other metrics are filtered out
     undef = maps:get(ReadsKey, Map, undef).
 
+get_metrics_by_name_empty_args(Config) ->
+    Result = get_metrics([], Config),
+    ParsedResult = get_ok_value([data, metric, getMetrics], Result),
+    lists:foreach(fun check_metric_by_type/1, ParsedResult).
```

Comment:
lists:foreach is not enough here.
You should also check that the list is empty or not. Same with foreaches below.

```erlang
get_metrics_by_name_empty_args(Config) ->
    Result = get_metrics([], Config),
    ParsedResult = get_ok_value([data, metric, getMetrics], Result),
    lists:foreach(fun check_metric_by_type/1, ParsedResult),
    [_|_] = ParsedResult.
```

or


```erlang
get_metrics_by_name_empty_args(Config) ->
    Result = get_metrics([], Config),
    ParsedResult = get_ok_value([data, metric, getMetrics], Result),
    lists:foreach(fun check_metric_by_type/1, ParsedResult),
    [] = ParsedResult.
```